cms/templates/visibility_editor.html

```diff
@@ -55,118 +50,51 @@ <h3 class="modal-section-title">${_('Make visible to:')}</h3>
+
+                                             </div>
                                         % endfor
-                                    % endfor
-
-                                    </div>
-                                </div>
-                            </section>
-                        </div>
-
-                        <div class="field field-radio field-visibility-level">
-                            <input type="radio" id="visibility-level-specific" name="visibility-level" value="" class="input input-radio visibility-level-specific"  ${'checked="checked"' if has_selected_content_groups else ''} />
-                            <label for="visibility-level-specific" class="label">${_('Specific Content Groups')}</label>
-                        </div>
-
-                        <div class="wrapper-visibility-specific">
-                    <section class="visibility-controls-secondary">
-                        <div class="visibility-controls-group">
-                            <h4 class="visibility-controls-title modal-subsection-title sr">${_('Content Groups')}</h4>
-                            <div class="list-fields list-checkbox">
-                            % for partition in cohort_partitions:
-                                % for group in partition["groups"]:
-                                    <div class="field field-checkbox field-visibility-content-group ${'was-removed' if group["deleted"] else ''}">
-                                        <input type="checkbox"
-                                            id="visibility-content-group-${partition["id"]}-${group["id"]}"
-                                            name="visibility-content-group"
-                                            value="${partition["id"]}-${group["id"]}"
-                                            class="input input-checkbox"
-                                            ${'checked="checked"' if group["selected"] else ''}
-                                        />
-                                        % if group["deleted"]:
-                                            <label for="visibility-content-group-${partition["id"]}-${group["id"]}" class="label">
-                                                ${_('Deleted Content Group')}
-                                                <span class="note">${_('Content group no longer exists. Please choose another or allow access to All Students and staff')}</span>
-                                            </label>
-                                        % else:
-                                            <label for="visibility-content-group-${partition["id"]}-${group["id"]}" class="label">${group["name"] | h}</label>
-                                        % endif
-                                    </div>
-                                % endfor
-                            % endfor
-
-                            ## Allow only one verification checkpoint to be selected at a time.
```

Comment:
Look @efischer19, I found more verification user partition changes I can delete! I'm removing some of the Studio code as part of my current work because I stumbled across it, and some of it I would need to modify if it stays in.